scalableminds/webknossos
# Activate user time tracking for non-admins

### URL of deployed dev instance (used for testing):
- https://allowuserstoseetimetracking.webknossos.xyz

### Steps to test:
-  I logged in as a normal user and verified that I can only see my own logs (I also send a curl request for another account id and didn't receive logs)
- I logged in as an admin and verified that I can see everything as usual

### Issues:
- fixes #3905 

------
- [x] wait for final "go" from the lab
